src/main/java/com/metamx/collections/bitmap/WrappedRoaringBitmap.java

```diff
@@ -126,6 +143,9 @@ public int size()
   @Override
   public void serialize(ByteBuffer buffer)
   {
+    if (compressRunOnSerialization) {
+      bitmap.runOptimize();
```

Comment:
@lemire runOptimize() returns a boolean, can you give more info about the returned value ? 
does a false returned value means it could not do the optimization ?